resources/blockchain/package-lock.json

```diff
@@ -7,7 +7,7 @@
       "resolved": "https://registry.npmjs.org/abstract-leveldown/-/abstract-leveldown-2.6.3.tgz",
       "integrity": "sha512-2++wDf/DYqkPR3o5tbfdhF96EfMApo1GpPfzOsR/ZYXdkSmELlvOOEAl9iKkRsktMPHdGjO4rtkBpf2I7TiTeA==",
       "requires": {
-        "xtend": "4.0.1"
+        "xtend": "~4.0.0"
```

Comment:
Do we need to go to an older version?

Reply:
please ignore this file. It supposed to be not checked-in

Reply:
@Dinesh2521 please undo this change and update the PR, will make it easier to merge.

Reply:
reverted the file to original state and checked in. Couldn't remove the file completely from PR.